tldraw/packages/react/src/components/Canvas/Canvas.tsx

```diff
@@ -110,6 +110,7 @@ export const Canvas = observer(function Renderer<S extends TLReactShape>({
   const selectedShapesSet = React.useMemo(() => new Set(selectedShapes || []), [selectedShapes])
   const erasingShapesSet = React.useMemo(() => new Set(erasingShapes || []), [erasingShapes])
   const singleSelectedShape = selectedShapes?.length === 1 ? selectedShapes[0] : undefined
+  const selectedOrHooveredShape = hoveredShape || singleSelectedShape
```

Comment:
It's a typo, right? Is it `selectedOrHoveredShape`?

Reply:
Indeed! Hoovered -> hovered 😅